app/editor/src/features/content/tool-bar/sections/filter/AdvancedSearchSection.tsx

```diff
@@ -66,7 +67,7 @@ export const AdvancedSearchSection: React.FC<IAdvancedSearchSectionProps> = ({
               options={[new OptionItem('', 0) as IOptionItem].concat([
                 ...filterEnabled(sourceOptions, filterAdvanced.searchTerm),
               ])}
-              value={sourceOptions.find((s) => s.value === filterAdvanced.searchTerm)}
+              value={sourceOptions.find((s) => s.label === filterAdvanced.searchTerm)}
```

Comment:
This looks like a bug to me.  It should be the value of an option that is selected, not the label.